src/components/BulkEnrollmentPage/stepper/BulkEnrollmentSubmit.test.jsx

```diff
@@ -0,0 +1,343 @@
+      <BulkEnrollmentSubmitWrapper
+        {...defaultProps}
+        bulkEnrollInfo={bulkEnrollWithAllSelectedRows}
+      />,
+    );
+    const button = screen.getByTestId(FINAL_BUTTON_TEST_ID);
+    userEvent.click(button);
+
+    const expectedParams = {
+      emails: userEmails,
+      course_run_keys: courseNames,
+      notify: true,
+    };
+
+    expect(LicenseManagerApiService.licenseBulkEnroll).toHaveBeenCalledWith(
+      defaultProps.enterpriseId,
+      expectedParams,
+    );
+    expect(logError).toBeCalledTimes(0);
+  });
+
+  it('tests notify toggle disables param to api service', () => {
+    const mockPromiseResolve = Promise.resolve({ data: {} });
+    LicenseManagerApiService.licenseBulkEnroll.mockReturnValue(mockPromiseResolve);
+
+    render(
+      <BulkEnrollmentSubmitWrapper
+        {...defaultProps}
+        bulkEnrollInfo={bulkEnrollWithAllSelectedRows}
+      />,
+    );
+    const button = screen.getByTestId(FINAL_BUTTON_TEST_ID);
+    const checkbox = screen.getByTestId(NOTIFY_CHECKBOX_TEST_ID);
+    userEvent.click(checkbox);
+    userEvent.click(button);
+
+    const expectedParams = {
+      emails: userEmails,
+      course_run_keys: courseNames,
+      notify: false,
+    };
+    expect(LicenseManagerApiService.licenseBulkEnroll).toHaveBeenCalledWith(
+      defaultProps.enterpriseId,
+      expectedParams,
+    );
+    expect(logError).toBeCalledTimes(0);
+  });
+
+  it('test component clears selected emails and courses after successful submit', async () => {
+    const mockPromiseResolve = Promise.resolve({ data: {} });
+    LicenseManagerApiService.licenseBulkEnroll.mockReturnValue(mockPromiseResolve);
+
+    render(
+      <BulkEnrollmentSubmitWrapper
+        {...defaultProps}
+        bulkEnrollInfo={bulkEnrollWithAllSelectedRows}
+      />,
+    );
+    const button = screen.getByTestId(FINAL_BUTTON_TEST_ID);
+    await userEvent.click(button);
+
+    expect(emailsDispatch).toBeCalledTimes(1);
+    expect(coursesDispatch).toBeCalledTimes(1);
+
+    expect(emailsDispatch).toHaveBeenCalledWith(
+      clearSelectionAction(),
+    );
+    expect(coursesDispatch).toHaveBeenCalledWith(
+      clearSelectionAction(),
+    );
+    expect(logError).toBeCalledTimes(0);
+  });
+
+  it('tests component creates toast after successful submit', async () => {
+    const mockPromiseResolve = Promise.resolve({ data: {} });
+    LicenseManagerApiService.licenseBulkEnroll.mockReturnValue(mockPromiseResolve);
+
+    render(
+      <BulkEnrollmentSubmitWrapper
+        {...defaultProps}
+        bulkEnrollInfo={bulkEnrollWithAllSelectedRows}
+      />,
+    );
+    const button = screen.getByTestId(FINAL_BUTTON_TEST_ID);
+    await userEvent.click(button);
+
+    expect(addToast).toBeCalledTimes(1);
+    expect(logError).toBeCalledTimes(0);
+    expect(addToast).toHaveBeenCalledWith(
+      `${userEmails.length} learners have been enrolled.`,
+    );
+  });
+
+  // TODO: figure out why the modal isn't shown :(
```

Comment:
is this a new ticket or just wip in this PR?

Reply:
Lael and I hit our collective heads against the proverbial wall and couldn't come up with a sufficient solution. We decided to leave it for a new ticket, unless you can come up with something!

Reply:
yeah cool as long as we ticket it up, thanks! if this works manually then yeah a test only ticket would be great

Reply:
everything works manually! The problem is that it's difficult to get the test/screen to recognize the new tab.